test/Token/ERC20/ZippieInternalTokenERC20.test.js

```diff
@@ -0,0 +1,330 @@
+const { BN, constants, expectEvent, expectRevert } = require("openzeppelin-test-helpers");
+const { expect } = require('chai');
+const ZippieInternalTokenERC20 = artifacts.require("ZippieInternalTokenERC20");
+
+contract("ZippieInternalTokenERC20", ([owner, sender, receiver, other]) => {
+
+  beforeEach(async function () {
+    this.token = await ZippieInternalTokenERC20.new("Token (XYZ)", "TOKEN-XYZ", 6, { from: owner });
+    this.token.mint(sender, new BN(1), { from: owner })
+  });
+
+  describe('ERC20 Whitelisted Transfer (WhitelistAdminRole)', function() {
```

Comment:
These whitelisted transfer related tests seem suitable for the issue, but why were all the ones below added also (or are they perhaps copied from somewhere into this file for some reason?)?

Reply:
Want to make sure that all other ERC20 functions also works as we expect, so we can detect errors if then imported OpenZeppelin contracts are changed (new versions)